### PR TITLE
blanket: update to 0.4.1

### PR DESCRIPTION
**Pull request by caughtquick**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/blanket/template [hunk 1]
```diff
@@ -9,6 +9,6 @@ short_desc="Improve focus and productivity by listening to different sounds"
 maintainer="Reback00 <reback00@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/rafaelmardojai/blanket"
-distfiles="https://github.com/rafaelmardojai/blanket/archive/${version}.tar.gz"
-checksum=79942ed919507c0078266d221c433ba9209f8ea5f11371d55d64bd3b469623bf
+distfiles="https://github.com/rafaelmardojai/blanket/archive/refs/tags/${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/blanket/template:12]: why did you change distfiles location?

Comment (caughtquick) [reply srcpkgs/blanket/template:12]: the old one wouldn't work when I tested it

Comment (paper42) [reply srcpkgs/blanket/template:12]: it works for me, can you try again?

Comment (caughtquick) [reply srcpkgs/blanket/template:12]: I am so sorry it does work,. let me fix that right away.

Comment (caughtquick) [reply srcpkgs/blanket/template]: I have fixed it and tested it and it still works

Comment (sgn) [reply srcpkgs/blanket/template]: While it does work, I think it won't work, soon. As of it is now, GitHub links to new format in the webpage.

Comment (caughtquick) [reply srcpkgs/blanket/template]: So should it in fact be the old fomat?

Comment (paper42) [reply srcpkgs/blanket/template]: oh, I am sorry for my wrong review

Comment (caughtquick) [reply srcpkgs/blanket/template]: Ok i'll just switch it back, I also fixed the trailing whitespace
